libcodechecker/libhandlers/analyze.py

```diff
@@ -177,6 +184,58 @@ def add_arguments_to_parser(parser):
                                help="File containing argument which will be "
                                     "forwarded verbatim for Clang-Tidy.")
 
+    if is_ctu_capable():
+        ctu_opts = parser.add_argument_group('cross translation unit analysis')
+        ctu_mutex_group = ctu_opts.add_mutually_exclusive_group()
+        ctu_mutex_group.add_argument('--ctu',
+                                     action='store_const',
+                                     const=[True, True],
+                                     dest='ctu_phases',
+                                     default=argparse.SUPPRESS,
+                                     help="Perform cross translation unit "
+                                          "(CTU) analysis (both collect and "
+                                          "analyze phases) using default "
+                                          "<ctu-dir> for temporary output. "
+                                          "At the end of the analysis, the "
+                                          "temporary directory is removed.")
```

Comment:
The concept of using temporary directories, at least outside of `quickcheck` has been eliminated earlier. @gyorb What do you think of a subcommand creating heavy temporary dumps and then removing them later on?

If the analysis fails, these files could be useful. Do we really want to clean up state that could help debugging? We don't clean failed analysis outputs either.

The option commands infer a lot of different responsibilities, they trigger multiple phases, decide on temporary folders, etc.